_posts/2023-09-10-august-rubygems-updates.md

```diff
@@ -0,0 +1,76 @@
+---
+title: August 2023 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in August.
+
+## RubyGems News
+
+This month in RubyGems, we released [3.4.18](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3418--2023-08-02), [3.4.19](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3419--2023-08-17) and Bundler [2.4.18](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2418-august-2-2023), [2.4.19](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2419-august-17-2023).
+
+
+In RubyGems, we improved the documentation performance by speeding up the build process when updating RubyGems - [#6864](https://github.com/rubygems/rubygems/pull/6864). We added a poller to fetch WebAuthn OTPs - [#6774](https://github.com/rubygems/rubygems/pull/6774), and removed some side effects when unmarshalling old `Gem::Specifications`- [#6825](https://github.com/rubygems/rubygems/pull/6825).
+
+Some other improvements that landed into our repo this month but that are not included in the above releases are:
+
+- optimized memory usage in `Bundler::Settings`, resulting in a faster boot time - [#6884](https://github.com/rubygems/rubygems/pull/6884).
+- added a `file` option to the Ruby method in the Gemfile- [#6876](https://github.com/rubygems/rubygems/pull/6876).
+- raised `Gem::Package::FormatError` when gem encounters corrupt EOF - [#6882](https://github.com/rubygems/rubygems/pull/6882).
+- resolved Ruby version file relative to bundle root - [#6892](https://github.com/rubygems/rubygems/pull/6892).
+- improved warning messages for bundled gems - [#6921](https://github.com/rubygems/rubygems/pull/6921).
+
+In August, RubyGems gained [109 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-08-01%7D...master@%7B2023-08-31%7D) contributed by 17 authors. There were 1,030 additions and 266 deletions across 97 files.
+
+## RubyGems.org News
+
+This month, RubyGems.org saw several bug fixes and updates, some of which include:
```

Comment:
`- fixed compact index files when gems are yanked https://github.com/rubygems/rubygems.org/commit/207be52ef6ce4fb9ee5eaed97c09f02277911da2`

Reply:
do we want to call out the launch of storing gem content files @martinemde ?

Reply:
Definitely should call out https://github.com/rubygems/rubygems.org/security/advisories/GHSA-rxcq-2m4f-94wm, that was a solid week+ of work for me

Reply:
> do we want to call out the launch of storing gem content files @martinemde ?

Yeah, turning that on is cool, though there's not much to see yet. Sure!

Reply:
> do we want to mention deploying the alpha release of OIDC support? https://github.com/rubygems/rubygems.org/pull/3716

I'd say no, because you said in that PR that you weren't planning on announcing the Alpha 😅 Seems like the html needs to be addressed before mentioning it.

Reply:
So I'll leave this one out then

> Definitely should call out [GHSA-rxcq-2m4f-94wm](https://github.com/rubygems/rubygems.org/security/advisories/GHSA-rxcq-2m4f-94wm), that was a solid week+ of work for me